Different integration times

hsergi opened the pull request:
Adding 3 levels of integration time for Roman CGI performance figure.

Commit:
Different integration times

Adding 3 levels of integration time for Roman CGI performance figure.

nasavbailey reviewed (changes requested):
Before merging back to main, original function call should still work. (ie: don't create a new plotting script). But the changes make sense overall. To try out new functions, I suggest creating a new branch, instead of main. In any case, thanks for these changes; I'll look at your fork to create my plot for my talk tomorrow.

Commit:
Bringing the changes to main

Reviewed labels and remove redundant files to merge the choice of different integration times with main.

ghost commented:
I reviewed labels and remove redundant files to merge the choice of different integration times with main. I did some extensive testing setting True/False for the CGI imager, band 3 spec and band 4 imager to make sure the individual labels and  the Roman CGI label appear at reasonable locations for each possible combination. I also updated the two YAML files in examples/ (default.yml and doc.yml). Please check it works as expected and the syntax in the functions is fine.

Commit:
Updated documentation, data and minor tweaks in plot_flux_ratio.py to adjust the position of the labels in some combinations of integration times

Added updated CGI Perf version, G0V stellar type as the reference one, and the three (optional) levels of integration times in documentation/description.md. Tested with printing out exposure times and without them. Updated data files to today's CGI Perf version.

Commit:
Minor tweak: 10,000hr->10000hr

Avoiding confusion with commas: 25, 100, 10,000hr-> 25, 100, 10000hr

Commit:
Minor tweaks in doc.yml

doc.yml: revert plotting CGI req to True, and reduce linewidth of Roman to 2.5 to improve the visualization when multiple iintegration times are displayed. Updated plot in documentation/

ghost commented:
Great. Thank you for the final suggestions.
